### PR TITLE
Upgrade Node 16 to 18 in documentation

### LINKED ISSUE #1035: Documentation for Node.js: 18 is now latest
Hi there, the [ReadMe for Node.js](https://github.com/GoogleContainerTools/distroless/blob/f0e17e79c46735364e4fd77ee4f0102a3a73ccce/nodejs/README.md) says that `latest` maps to `16` to mirror the [official node image](https://hub.docker.com/_/node). `18` is now `latest` so distroless should update its docs and tagging to reflect this.

### PR DESCRIPTION
**Pull request by luc122c**
Closes #1035

### CODE REVIEW

File: examples/nodejs/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
-FROM node:16 AS build-env
+FROM node:18 AS build-env
 COPY . /app
 WORKDIR /app
 
-RUN npm ci --only=production
+RUN npm ci --omit=dev
```
Comment (loosebazooka) [inline examples/nodejs/Dockerfile:5]: Sorry, not super familiar with node, what's the difference here between these two options?

Comment (luc122c) [reply examples/nodejs/Dockerfile:5]: When I was testing the changes, I got a warning in the console from NPM. Apparently `--only` [has been deprecated](https://docs.npmjs.com/cli/v8/using-npm/config#only).
Screenshot 2022-05-09 at 21 11 48

Comment (loosebazooka) [reply examples/nodejs/Dockerfile:5]: sgtm

**Review by loosebazooka — APPROVED** [review]

**Comment by loosebazooka** [comment]
thanks!

**Pull request merged**